scripts/do-release.sh

```diff
@@ -203,7 +203,16 @@ function update_operator_pr() {
         update_go_mod "${target}"
     done
 
-    sed -i -E "s/(.*Version +=) .*/\1 \"${release['version']#v}\"/" "projects/${project}/apis/submariner/v1alpha1/versions.go"
+    local versions_file="projects/${project}/api/submariner/v1alpha1/versions.go"
+    if [[ "${semver['major']}" -eq 0 ]]; then
+        if [[ "${semver['minor']}" -eq 11 ]]; then
+            versions_file="projects/${project}/apis/submariner/v1alpha1/versions.go"
+        elif [[ "${semver['minor']}" -lt 11 ]]; then
+            versions_file="projects/${project}/pkg/versions/versions.go"
+        fi
+    fi
```

Comment:
Wouldn’t it be better to check for the file’s existence, rather than hard-coding versions?

Reply:
I changed it to find the file instead